### ISSUE TITLE
Showing characters are not clothed when they are

### ISSUE THREAD

**Issue by WhackChidori** [dalaranwow/dalaran-wow#4880]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
When playing on my character and looking at other characters, it shows other players not clothed showing them naked with just their weapons but NPC's are clothed and when my friend who also has an account logged into his account on my computer, his character and account did not have this issue. I've restarted my computer, I've reloaded the UI in wow a million times and the only thing I haven't tried is re-downloading the client.
**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. Just every time I log in the result is the same no matter what I do
2. Someone would just have to see it through my account
3.  I honestly dunno how to reproduce issue with someone else but just need help :(

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Thank you for the report @WhackChidori .
Regarding the issue I am not sure if it's server or client bug as it happens randomly even now in retail legion 7.3 within areas with a lot of people.

**Comment by Rushor** [comment]
duplicate of https://github.com/dalaranwow/dalaran-wow/issues/3826
